libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -511,10 +491,8 @@ class AC_AttitudeControl {
     // tail rotor thrust in hover. Overloaded by AC_Attitude_Heli to return angle.
     float get_roll_trim_rad() { return cd_to_rad(get_roll_trim_cd()); }
 
-    // passthrough_bf_roll_pitch_rate_yaw - roll and pitch are passed through directly, body-frame rate target for yaw
-    // this assumes a maximum deflection rate of 45 degrees per second or pi/4 rad/s.
-    void passthrough_bf_roll_pitch_rate_yaw_cds(float roll_passthrough_cds, float pitch_passthrough_cds, float yaw_rate_bf_cds);
-    virtual void passthrough_bf_roll_pitch_rate_yaw_rads(float roll_passthrough_rads, float pitch_passthrough_rads, float yaw_rate_bf_rads) {};
+    // passthrough_bf_roll_pitch_rate_yaw_norm - roll, pitch and yaw passed through directly to the motor mixers
+    virtual void passthrough_bf_roll_pitch_rate_yaw_norm(float roll_passthrough_norm, float pitch_passthrough_norm, float yaw_passthrough_norm) {};
```

Comment:
I wonder if it would build if this was pure virtual.
```suggestion
    virtual void passthrough_bf_roll_pitch_rate_yaw_norm(float roll_passthrough_norm, float pitch_passthrough_norm, float yaw_passthrough_norm) = 0;
```

Reply:
The advantage of pure virtual is that if someone tries to use it on a multi-rotor it will result in a build error, currently it will build but it won't do what the user is expecting.

But its possible that the compiler is not smart enough to work out that its not used except by heli so it already won't build.

Reply:
Yes, I just changed the units.

Reply:
Yeah not a blocker, Just spotted in passing.